docs/book/modules.md

```diff
@@ -0,0 +1,101 @@
+# Modules
+
+A module is a PHP namespace. It can contain controllers, classes, view scripts, configuration, tests, as well as public assets such as images, CSS, and JavaScript. For example, if you have an e-commerce application, you could have a module for presenting and selling products, and another for administrators to create products and manage orders.
```

Comment:
```suggestion
A module is a package of logically grouped resources in PHP namespace. It can contain controllers, classes, view scripts, configuration, tests, as well as public assets such as images, CSS, and JavaScript. For example, if you have an e-commerce application, you could have a module for presenting and selling products, and another for administrators to create products and manage orders.

Strictly speaking, for the application module is defined by the class `Module` in module namespace which is responsible for providing configuration necessary to setup the application. No assumptions are made about file or code structure. 
```

Reply:
There are two conflated concepts. A module that is a package in `module/` and a module that is configuration provider referenced from modules list.